frontend/js/src/metadata-viewer/MetadataViewerPageWrapper.tsx

```diff
@@ -128,6 +128,20 @@ document.addEventListener("DOMContentLoaded", () => {
 
   const { playing_now } = reactProps;
 
+  /** Fetch and save list of MusicBrainz genres on startup
+   * Not sure where this should end (list of genres sent by back-end instead?)
+   * but for now genres are only used on this page
+   */
+  try {
+    const response = await fetch(
+      "https://musicbrainz.org/ws/2/genre/all?fmt=txt"
+    );
+    const genresList = await response.text();
+    globalAppContext.musicbrainzGenres = Array.from(genresList.split("\n"));
+  } catch (error) {
+    console.error(error);
+  }
+
```

Comment:
In a follow-up PR, we should probably load those as props from the back-end, and cache them aggressively?

Reply:
Sounds good to me, Maybe a ticket or ping me tomorrow and I'll add it.